### PR TITLE
Various fixes for LBA1 and LBA2

### PR DESCRIPTION
**Pull request by xesf**
LBA2 and LBA1:
* Fixed platform GOTO_3D_POINT scripting command
* Fixed Anim actions for LBA1 and LBA2
* Behaviour Menu inventory flag checks

LBA1 only:
* Enabling Extras for LBA1
* Fixed Actor Collisions for LBA1

Apart for some weird bugs in the scripts and also with the animation actions, LBA1 is very compatible.

**Preview here:** https://pr-556.lba2remake.net

### CODE REVIEW

File: src/resources/parsers/entity1.ts [hunk 1]
```diff
@@ -189,24 +190,26 @@ const loadEntityAnim = (data, offset) => {
                 case ACTIONTYPE.SAMPLE_REPEAT:
                     action.sampleIndex = data.getUint16(innerOffset + offset + 2, true);
                     action.repeat = data.getUint16(innerOffset + offset + 4, true);
-                    innerOffset += 10;
+                    // innerOffset += 10;
```
Comment (slwilliams) [inline src/resources/parsers/entity1.ts:193]: remove?

File: src/resources/parsers/entity1.ts [hunk 2]
```diff
@@ -189,24 +190,26 @@ const loadEntityAnim = (data, offset) => {
...
-                    action.unk2 = data.getUint8(innerOffset + offset + 5, true);
-                    action.unk3 = data.getUint16(innerOffset + offset + 7, true);
-                    action.unk4 = data.getUint8(innerOffset + offset + 8, true);
-                    innerOffset += 8;
+                    action.yHeight = data.getUint16(innerOffset + offset + 2, true) * WORLD_SCALE;
+                    action.unk1 = data.getInt8(innerOffset + offset + 4, true);
+                    action.unk2 = data.getInt8(innerOffset + offset + 5, true);
+                    action.unk3 = data.getInt16(innerOffset + offset + 7, true);
+                    action.unk4 = data.getInt8(innerOffset + offset + 8, true);
+                    // innerOffset += 8;
```
Comment (slwilliams) [inline src/resources/parsers/entity1.ts:202]: remove

**Review by slwilliams — APPROVED** [review]

File: src/resources/parsers/entity1.ts (hunk 1 above)
Comment (xesf) [reply src/resources/parsers/entity1.ts]: done

File: src/resources/parsers/entity1.ts (hunk 2 above)
Comment (xesf) [reply src/resources/parsers/entity1.ts]: done

**Review by agrande — APPROVED** [review]

**Comment by agrande** [comment]
This makes it possible to go out of the citadel, nice!

**Pull request merged**